README.md

```diff
@@ -1,87 +1,71 @@
 # Home Unite Us
 
-We're working with community non-profits who have a host home initiative to develop a workflow management tool to make the process scaleable (across all providers), reduce institutional bias, and effectively capture data.
+Home Unite Us is a community developed web platform designed to support and scale the [host homes](https://www.pointsourceyouth.org/interventions/host-homes) initiative.
 
-Host home programs are centered around housing young people, 18 - 25 years old. Their approach focuses on low-cost, community-driven intervention by matching a willing host with a guest or group of guests, providing a stable housing environment for youths who are experiencing homelessness and seeking stable housing.
+Host homes provide short-term housing interventions for young people, aged 18-25, who are experiencing homelessness for any variety of reasons. The process for matching youth with a temporary safe space can be challenging, but numerous organization have implemented programs tailored to the needs of their local area. These implementations can vary in their exactly policies and procedures, but they are all driven by the core principles of the host homes initiative and they often share an overlapping set of technological needs.
```

Comment:
"... in their *exact..."

Reply:
Thanks! Corrected in [ebfba50](https://github.com/hackforla/HomeUniteUs/pull/557/commits/ebfba503f84ed549893cd2f68623dc90597fdafe)